sentry-ruby/lib/sentry/configuration.rb

```diff
@@ -395,6 +401,18 @@ def enable_tracing=(enable_tracing)
       @traces_sample_rate ||= 1.0 if enable_tracing
     end
 
+    def profiles_sample_rate=(profiles_sample_rate)
+      log_info("Thank you for trying out Profiling on Sentry. Please note that this is currently an experimental feature.")
```

Comment:
I feel that noting this is experimental in docs and changelog should be enough. If I were an user that opt in to this voluntarily, these information wouldn't be helpful.

We should keep the `stackprof` inclusion warning though.